controllers/managedocs_controller.go

```diff
@@ -1362,6 +1358,37 @@ func (r *ManagedOCSReconciler) reconcileOCSCSV() error {
 	return nil
 }
 
+func (r *ManagedOCSReconciler) reconcileMCGOperatorCSV() error {
+	csvList := opv1a1.ClusterServiceVersionList{}
+	if err := r.list(&csvList); err != nil {
+		return fmt.Errorf("unable to list csv resources: %v", err)
+	}
```

Comment:
Having the list here will basically list all CSV twice in a single iteration. 
I would prefer not to do so. Can we move this to a setup state that lists all CSVs and then reconcile each based o the existing list?